python/cugraph/cugraph/dask/traversal/bfs.py

```diff
@@ -122,12 +122,28 @@ def bfs(input_graph,
 
     if not isinstance(start, (dask_cudf.DataFrame, dask_cudf.Series)):
         if not isinstance(start, (cudf.DataFrame, cudf.Series)):
-            start = cudf.Series(start)
+            vertex_dtype = input_graph.nodes().dtype
+            start = cudf.Series(start, dtype=vertex_dtype)
         if isinstance(start, (cudf.DataFrame, cudf.Series)):
             # convert into a dask_cudf
             start = dask_cudf.from_cudf(start, input_graph._npartitions)
```

Comment:
Seems like we didn't need the `if isinstance` block, or am I missing something?
```suggestion
        # convert into a dask_cudf
        start = dask_cudf.from_cudf(start, input_graph._npartitions)
```

Reply:
Right. I am missing and `else` statement here which should raise a type error.

```
else:
     raise TypeError("'start' must be a list or a cudf object or a dask 
                     f"object, got :  {type(start)}")
```

Reply:
Right. This extra check is unnecessary